Makefile.am

```diff
@@ -3,7 +3,7 @@ ACLOCAL_AMFLAGS = -I m4
 
 UAP_REGEX_FILE=regexes.yaml
 
-SUBDIRS=ext docs docker
```

Comment:
this might be a bad idea. it's probably a lot better idea to use EXTRA_ in the docker/Makefile.am for the targets, so they don't get built automatically.